fluss-lake/fluss-lake-iceberg/src/main/java/com/alibaba/fluss/lake/iceberg/IcebergLakeCatalog.java

```diff
@@ -145,10 +143,11 @@ private void createTable(TablePath tablePath, Catalog.TableBuilder tableBuilder)
         }
     }
 
-    public Schema convertToIcebergSchema(TableDescriptor tableDescriptor) {
+    public Schema convertToIcebergSchema(TableDescriptor tableDescriptor, boolean isPkTable) {
         List<Types.NestedField> fields = new ArrayList<>();
         int fieldId = 1;
```

Comment:
Could you modify `int fieldId = 1;` to `int fieldId = 0;` for I just found that flink/spark all from `fieldId=0` to create iceberg table field.